lib/autoform.ex

```diff
@@ -76,6 +110,56 @@ defmodule Autoform do
           [conn, action, opts]
         )
       end
+
+      defp put_fields(assigns, schema, options) do
+        excludes = Keyword.get(options, :exclude, []) ++ [:id, :inserted_at, :updated_at]
```

Comment:
smart way to exclude auto-gen fields and allow people to customise. as always. good code!